### ISSUE TITLE
I can't access the game's start screen. Please help me.

### ISSUE THREAD

**Issue by ken27dwman** [KC3Kai/KC3Kai#2759]
### Versions: 
Chrome 72.0.3626.81 (64bit) / Windows 10 (64bit) / KC3 33.7.1
I guess my KC3 is webstore version. (I installed KC3 through chreome webstore)
And I play game in [ Toggle Features+Apply Customizations & Customize DMM Website ] mode

### Steps to reproduce:
I don't know how to say. 
It happens randomly when I start a game.
(Game connection problem)

### Actual behaviour:
I've been using KC3 well, but nowadays I've been having a game connection problem.
If I play a game through KC3, I can't access the game's start screen. 
Only I can get white screen(it just stopped and never load anything), or gray screen(with a strange square face & Error message)
I've tried everything I can do, but it's not working.
(clear the cache, reinstall KC3, reinstall the chrome, reinstall the Windows 10 etc.)
Please help me.

### Expected behaviour:
I don't know what to say. 
I just want to play game.

### Screenshots (if applicable):
I have screenshots of white screen and gray screen. 
But I can't find upload button.

### Error Report (if applicable):
I have no JSON files.

**Comment by ken27dwman** [comment]
![case 1](https://user-images.githubusercontent.com/47293930/52481640-63536300-2bf2-11e9-9d34-8e09781b4a98.png)
![case 2](https://user-images.githubusercontent.com/47293930/52481644-651d2680-2bf2-11e9-949d-955aabb1b34f.png)

This is what I got. AND I am STILL experiencing this problem.
Even today.
plz help

**Comment by sorewachigauyo** [comment]
Is there anything in the console tab and are you using a VPN/proxy to connect to the game?

**Comment by ken27dwman** [comment]
> Is there anything in the console tab and are you using a VPN/proxy to connect to the game?

I'm not using any VPN or Proxy. 
Just my computer and chrome and KC3 only.
I don't understand how this error occurs. 
I just reinstalled chrome and kc3, this time it works.(But I got 4 red icons and 2 yellow icons)
I will check when this error occurs again. 
Should I just copy and paste it? Or should I take a screenshot?

**Comment by sorewachigauyo** [comment]
Oh, if there's no further problem on your end then there's no need.

**Comment by ken27dwman** [comment]
> Oh, if there's no further problem on your end then there's no need.

This issue happened again.
This time I captured the contents of the console tab.
However I'm concerned that this may contain my personal information.
Is it safe to post on here?
Anyway I post part of it.
![case3-1](https://user-images.githubusercontent.com/47293930/52521970-b72d7d00-2cc2-11e9-8b1a-0b8b14495d39.png)

**Comment by ken27dwman** [comment]
This time even reinstalling doesn't work. 
I really need help.
ps- I suffered this problem almost a month.

**Comment by DynamicSTOP** [comment]
@ken27dwman 
> ![case3-1](https://user-images.githubusercontent.com/47293930/52521970-b72d7d00-2cc2-11e9-8b1a-0b8b14495d39.png)

That looks like ads error that you are showing now.  But judging from screenshot there should be 7 more errors besides that one. 
I don't use adblock on profile with kc3 and i don't have that particular error, so i suppose it's related to you connection. Consider checking you adblock\antivirus software (some of them like to interfere with web traffic).

You can also try to play without kc3 extension and see if it loads up normally.

**Comment by sinsinpub** [comment]
Since KC phase 2, everyone should aware that HTML5 ver. game depends on the following 3rd-party open-source javascript libraries:
 * [axios](https://www.google.com/search?q=axios) hosted at `https://unpkg.com/axios/dist/axios.min.js`
 * [tweenjs](https://www.google.com/search?q=tweenjs) hosted at `https://code.createjs.com/tweenjs-0.6.2.min.js`
 * [pixi.js](https://www.google.com/search?q=pixi.js) hosted at `https://cdnjs.cloudflare.com/ajax/libs/pixi.js/4.8.1/pixi.min.js`
 * [howler.js](https://www.google.com/search?q=howler.js) hosted at `https://cdnjs.cloudflare.com/ajax/libs/howler/2.0.15/howler.core.min.js`

http request failure on any of these files will cause game error or stuck at startup loading phase. check ur console's error messages, if any of these files appear in red, try to find out why they are not available for your network.
possible reasons: just unstable network connection; ad block/anti-virus software mentioned above; cdn trouble for your area; affected/blocked by your ISP/country

**Comment by ken27dwman** [comment]
First of all, thanks to everyone who helped me.
But I think I can't solve this problem.
I don't use any adblock apps, and I use window basic anti-virus program only.(window 10's window defender)
I reset my internet modem, but it didn't work.
I lowered window defender's web security, but it didn't work.
Finally, I bought VPN (japan IP location) and tried to connect game, but now DMM is blocking my connection because of 'unauthorized access'.
I tried everything I could. But there seems to be no more method....

**Comment by sorewachigauyo** [comment]
If you're using VPN, you should not be getting the unauthorized access page. (That usually applies to connection outside japan, so maybe your VPN isn't working correctly)

If you're not using VPN, check that you're on the latest version of kc3 (33.7.2) and enable cookie hack. As @DynamicSTOP has mentioned, you may want to let us see the 7 remaining errors to see what's wrong.

**Comment by ken27dwman** [comment]
![case3-2](https://user-images.githubusercontent.com/47293930/52567835-9135e300-2e50-11e9-9d97-99a474af0f1f.png)
#1 errors
Okay. I'll post full screen shot.
I deleted the detailed address because of the personal information leak.
(I'm sorry for being so sensitive.)
If you need full address information, please let me know your personal email address.
Or tell me how to send a private message.
I'll send it to you.

![ask 1](https://user-images.githubusercontent.com/47293930/52567843-97c45a80-2e50-11e9-8c67-7622904c36b3.PNG)
#2 cookie hack
You mean this?
I just discovered that I haven't checked this since the KC second phase update.
I really don't understand.
I have been played the game without checking this item until this disaster happens.
But there was not a big problem loading the start screen, so why is this happening now?
And is it safe to check this item?
Is there any danger of BAN?

**Comment by sorewachigauyo** [comment]
Cookie hack is to bypass DMM's region lock, if you have no issue with that, then don't worry about it.

It seems like you are unable to access some game data for some reason, are you running an adblocker? Also, which region are you from?

**Comment by ken27dwman** [comment]
> Cookie hack is to bypass DMM's region lock, if you have no issue with that, then don't worry about it.
> It seems like you are unable to access some game data for some reason, are you running an adblocker? Also, which region are you from?

As I said before, I have never been used any adblocker programs.

Anyway, is it safe to use Cookie hack?
As you advised, I used Cookie hack and tried again.
Unfortunately, I'm still experiencing connection error.
It is difficult to say under which conditions the error occurs, because it occurs randomly.

I didn't know the reason of this problem until now, but now I guess I know why.
I access Kancolle from 'Korea' region.
I think this is the reason.

**Comment by ken27dwman** [comment]
Blocking access to oversea servers started from february 11.
If there was an advance preparation process before the action, It explains why the connection has been in trouble about a month ago.
If my guess is right, I am really sorry for everyone who helped me.

**Comment by ken27dwman** [comment]
Hi. I have been tried all the methods that many friends on this post have advised to me.
I have heard that the Korean government blocked DMM game servers and released it again, but I don't how it is going on now.
(There is no announcement about withdrawing SNI field interception yet.)
In the meantime, I have blocked all security program firewalls and I called the ISP and changed my router.
Fortunately, gray screen does not appear now, but white screen often comes out.
Anyway, if I delete all cache files before starting game, it is connected without error.
Of course I can't delete all cache files everytime forever, but I think I should play like this for a while.
I don't think there is any better solution for me.
I should close this post for now, and if I have another problem, I'll post new post.
I've been very grateful to all of you.
